add a note recommending git-secrets for scanning (#5139)
aws-amplify/docs: src/pages/cli/start/workflows.mdx

- `amplify <category> update`
- `amplify <category> remove`
- `amplify <category> push`

## Secrets scanning

> When working with secrets in your Amplify project it is important to safeguard your secrets from being exposed by being committed to GitHub. We recommend using [git-secrets](https://github.com/awslabs/git-secrets) from [AWS Labs](https://github.com/awslabs). Once installed, it will scan your local repository for secrets and prevent you from committing them.

Reviewer: I think this can be generalized a bit since we don't require git as the VCS.

```suggestion
When working with secrets in your Amplify project it is important to safeguard your secrets from being committed to version control. If you use git, we recommend using [git-secrets](https://github.com/awslabs/git-secrets) from [AWS Labs](https://github.com/awslabs). Once installed, it will scan your local repository for secrets and prevent you from committing them.
```

Author: Makes sense to me!
